packages/frontend/src/PortalMap/index.tsx

```diff
@@ -108,10 +109,12 @@ const PortalMap = () => {
       }
 
       if (t.isNode()) {
+        console.log(t.data())
         const name = t.data('zoneName')
         const id = t.data('zoneId')
 
-        dispatch({ type: PortalMapActionTypes.INSPECT, inspectId: id })
+        dispatch({ type: PortalMapActionTypes.CLEARINSPECT })
+        dispatch({ type: PortalMapActionTypes.INSPECT, inspectFromId: id })
```

Comment:
So here, I would write a new reducer/action for the new type of INSPECT you're doing and clear it that way. Doing it this way you're not guaranteed sequence and it could get weird. Best to just do what you want in your own special action.

Reply:
actually, I'm not sure what this is doing. I thought this was yours but this was one of mine. Mine should be doing that in the reducer on that action already.

Reply:
It wasn't, because the `useEffect()` that should change the From and To selector in the UI wasn't doing anything since it was checking that the Ids weren't null, but the CLEARINSPECT was setting them to null.